src/js/media-chrome-button.js

```diff
@@ -11,15 +11,15 @@ template.innerHTML = `
     justify-content: center;
     vertical-align: middle;
     box-sizing: border-box;
-    background: var(--media-control-background, rgba(20,20,30, 0.7));
+    background: var(--media-control-background, var(--media-secondary-color, rgb(20 20 30 / 0.7)));
 
     padding: var(--media-control-padding, 10px);
 
     ${/* Vertically center any text */''}
     font-size: 14px;
     line-height: var(--media-text-content-height, var(--media-control-height, 24px));
     font-weight: bold;
-    color: #fff;
+    color: var(--media-control-font-color, var(--media-primary-color, rgb(238 238 238)));
```

Comment:
- Adding CSS variable for consistency, with standard composition, including defaults.

Reply:
🎉